content/nomos/updates/2024-02-05.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 2024-02-05 Nomos weekly
+tags:
+  - nomos-updates
+date: 2024-02-05
+lastmod: 2024-02-06
+draft: false
+description: Weekly update of Nomos
+---
+## `network privacy and mixnet`
+
+### `research`
+
+- Polishing the spec regarding the "robustness layer" - we've had some [concerns](https://www.notion.so/Mixnet-Specification-807b624444a54a4b88afa1cc80e100c2?d=80f00df5f08843bd95d7cf3552bad4c1#8aec5ec23da548388ce89e56ba21cd9f) about it, so there are several ongoing discussions on [Notion](https://www.notion.so/Mixnet-Specification-807b624444a54a4b88afa1cc80e100c2?d=7aaa6b805985476794a33359d21660b4#e041fafe4de54efd9ea142e551f1fc48) and [GitHub](https://github.com/logos-co/nomos-specs/pull/57#discussion_r1472490489) in regards to answering them. More precisely, by this [PR](https://github.com/logos-co/nomos-specs/pull/61) (in agreement so far but awaiting final review) there is a suggestion not to define the robustness layer in the specification explicitly because it may need to be an implementation detail. If this is accepted, we're able to remove the confusing term "layer" from the specification.
```

Comment:
As I mentioned to Youngjoon, we should not call this "robustness layer". It's not a "robustness protocol" and is not a layer. It's rather a module that provides configuration, and as we agreed later on, it's an implementation detail.